easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,38 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'c-ares'
+version = '1.18.1'
+
+homepage = 'https://github.com/c-ares/c-ares'
+description = """This is c-ares, an asynchronous resolver library. 
+It is intended for applications which need to perform DNS queries 
+without blocking, or need to perform multiple DNS queries in parallel. 
+The primary examples of such applications are servers which communicate 
+with multiple clients and programs with graphical user interfaces."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/c-ares/c-ares/archive']
+sources = ['cares-%s.tar.gz' % version.replace('.', '_')]
+
+checksums = [
+    '414872549eec4e221b576693fdc9c9bce44ff794d0f1f06f2515b56a7f6ec9c9',  # cares-1_18_1.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('pkg-config', '0.29.2'),
+    ('CMake', '3.22.1'),
+]
+
+parallel = 2
+
+sanity_check_paths = {
+    'files': ['lib/libcares.%s' % SHLIB_EXT],
+    'dirs': ['include', 'share'],
```

Comment:
Existing easyconfigs have a slightly more extensive set of paths to check, please sync up?